mailpoet/assets/js/src/segments/dynamic/templates/templates.tsx

```diff
@@ -74,30 +55,11 @@ export const templates: SegmentTemplate[] = [
     filters: [
       {
         segmentType: 'userRole',
-        action: 'lastClickDate',
-        operator: 'inTheLast',
-        value: '90',
-      },
-      {
-        segmentType: 'userRole',
-        action: 'lastOpenDate',
-        operator: 'inTheLast',
-        value: '90',
-      },
-      {
-        segmentType: 'userRole',
-        action: 'lastPurchaseDate',
-        operator: 'inTheLast',
-        value: '90',
-      },
-      {
-        segmentType: 'userRole',
-        action: 'lastSendingDate',
+        action: 'lastEngagementDate',
         operator: 'inTheLast',
         value: '90',
       },
     ],
-    filtersConnect: SegmentConnectTypes.OR,
   },
   {
     name: __('Engaged Subscribers (6 months)', 'mailpoet'),
```

Comment:
The ticket mentions changing three segment templates: `Engaged Subscribers (30 days)`, `Engaged Subscribers (3 months)`, and `Engaged Subscribers (6 months)`. I think you also need to change the template for the 6 months.

Reply:
Thanks for catching this, I can't believe I missed it 🤦

Updated in e610fe3cefc20166c0584c399719e550a77f9803